FrontEnd/src/views/DashboardView.vue

```diff
@@ -1,5 +1,73 @@
 <template>
-  <div class="about">
-    <h1>Dashboard under development</h1>
+  <div id="x-main" class="container-full p-3">
+<div class="container-md">
+
+	<div class="row">
+	<div class="col-12 p-2">
+		<h3 class="float-start ps-2">Malnutrition Dashboard (Admin)</h3>
+	</div>
+
+	<div class="map-container">
+    <l-map :zoom="leafletdata.zoom" :center="leafletdata.center">
+      <l-tile-layer :url="leafletdata.url"></l-tile-layer>
+      <l-marker :lat-lng="leafletdata.marker"></l-marker>
+      <l-popup :lat-lng="leafletdata.marker">
+        <p>Hello, I'm a popup</p>
+      </l-popup>
+    </l-map>
   </div>
+
+
+	<div class="col-4" style="min-height:420px">
+		<div class="card mb-4 rounded-3 shadow-sm">
+          <div class="card-header">
+            <h4 class="my-0 fw-normal">472 Aanganwadis</h4>
+          </div>
+          <div class="card-body">
+            <h3 class="card-title pricing-card-title">2,306 <small class="text-muted fw-light">[SAM] <i class="bi bi-arrow-up-circle text-danger"></i> 37 </small></h3>
+            <h3 class="card-title pricing-card-title">7,812 <small class="text-muted fw-light">[MAM] <i class="bi bi-arrow-down-circle text-success"></i> 123 </small></h3>
+          </div>
+        </div>
+
+		<div class="card rounded-3 shadow-sm" style="min-height:220px">
```

Comment:
Could you please address all the alignment issues in this files and extra lines where ever not required can be removed.

Reply:
changes updated
Map is not implemented properly please check this issues